docs/content/latest/api/ysql/datatypes/type_datetime/date-time-data-types-semantics/type-interval/interval-arithmetic/_index.md

```diff
@@ -13,21 +13,28 @@ isTocNested: true
+You need to understand the notions that the section [Two ways of conceiving of time: calendar-time and clock-time](../../../conceptual-background/#two-ways-of-conceiving-of-time-calendar-time-and-clock-time) addresses in order to understand the code and the explanations in this page's child page [The moment-_interval_ overloads of the "+" and "-" operators for _timestamptz_, _timestamp_, and _time_](./moment-interval-overloads-of-plus-and-minus/). The notions help you understand how the semantic rules of the native [moment-moment overloads of the "-" operator for timestamptz, timestamp, and time](./moment-moment-overloads-of-minus/) are ultimately confusing and therefore unhelpful—and why you should therefore adopt the practices that the section [Custom domain types for specializing the native _interval_ functionality](../custom-interval-domains/) explains. The distinction between _clock-time-semantics_ and _calendar-time-semantics_ is only implicitly relevant for the notions that the sections [The interval-interval overload of the "=" operator](#the-interval-interval-overload-of-the-operator) and [Interval-only addition/subtraction and multiplication/division](#interval-only-addition-subtraction-and-multiplication-division) explain.
 
 The PostgreSQL documentation does not carefully specify the semantics of _interval_ arithmetic. This page and its children aim to specify the operations in terms of the individual fields of the [internal representation](../interval-representation/).
 
-{{< note title="The results of interval arithmetic are, in general, sensitive to the session's timezone." >}}
-More carefully stated, the results of some moment-interval arithmetic operations (the moment-moment overloads of the `-` operator and the moment-interval overloads of the `+` and `-` operators) are sensitive to the session's _TimeZone_ setting when the moments are _timestamptz_ values.
+{{< note title="The results of 'interval' arithmetic are, in general, sensitive to the session's timezone." >}}
```

Comment:
Straight quotes are almost never correct in text, and in this case you can just dispense with the quotes entirely.
```suggestion
{{< note title="The results of interval arithmetic are, in general, sensitive to the session's timezone." >}}
```